docs/guide/http_cache.md

```diff
@@ -88,127 +191,43 @@ services:
-
-If the content you're rendering depends on a user's permissions, then you should make the response context-aware:
-
-``` php
-use Symfony\Component\HttpFoundation\Response;
- 
-// Inside a controller action
-// Tells proxy configured to support this header to take the rights of a user (user hash) into account for the cache
-$response = new Response();
-$response->setVary('X-User-Hash');
-```
-
-## Configuration
-
-### Cache Purge
-
-This page explains the content cache purge *(aka invalidate)* mechanism used when publishing content from the UI or from a container-aware script, resulting in cache being invalidated either in the built-in Symfony Reverse Proxy, or on the much faster Varnish reverse proxy.
+## Cache purging
 
-Note that if you use a non-reverse proxy that does not support purge headers, shared content will stay in the cache for the whole duration defined by s-maxage, without the possibility of clearing it.
+The Content cache purge (invalidate) mechanism is used when publishing content from the UI or from a container-aware script
+This results in cache being invalidated either in the built-in Symfony reverse proxy, or on the much faster Varnish reverse proxy.
 
-#### Overview
+Note that if you use a non-reverse proxy that does not support purge headers,
+shared content will stay in the cache for the whole duration defined by `s-maxage`,
+without the possibility of clearing it.
 
-eZ Platform returns content-related responses with an `X-Location-Id` header that are stored together by the configured HTTP cache. This allows you to clear *(invalidate)* HTTP cache representing specifically a given Content item. On publishing the content, a cache purger is triggered with the Content ID in question, which in turn figures out affected content Locations based on [Smart HTTP cache clearing](#smart-http-cache-clearing) logic. The returned Location IDs are sent for purge using the purge type explained further below.
+eZ Platform returns content-related responses with an `X-Location-Id` header.
+The responses are stored together by the configured HTTP cache.
+This allows you to clear (invalidate) HTTP cache representing specifically a given Content item.
+On publishing the Content, a cache purger is triggered with the Content ID in question,
+which in turn figures out affected Locations based on [smart HTTP cache clearing](#smart-http-cache-clearing) logic.
+The returned Location IDs are sent for purge using the selected purge type.
 
-#### Purge types
+### Purge types
 
-##### Symfony Proxy: Local purge type
+#### Symfony Proxy: Local purge type
 
-By default, invalidation requests will be emulated and sent to the Symfony Proxy cache Store. Cache purge will thus be synchronous, meaning no HTTP purge requests will be sent around when publishing.
+By default, invalidation requests will be emulated and sent to the Symfony proxy cache store.
+Cache purge will be synchronous, meaning no HTTP purge requests will be sent around when publishing.
```

Comment:
Update: Okay, I read this again now.
So we are in the context of `local purge type`. And it might actually be correct that no purge requests are sent in 1.7 ( but that is not the case for 1.13, ref https://github.com/ezsystems/ezplatform-http-cache/blob/v0.6.0.1/src/PurgeClient/LocalPurgeClient.php#L58 )
Those purge requests in 1.13 will still be handled synchronously though, as stated